docs/index.html

```diff
@@ -297,11 +372,11 @@ <h2>#TwitterThrowback</h2>
     /**
      * @param {string} username
      */
-    function getTweetsOnThisDay(username) {
+    function getTweetsOnThisDay(username, day, month) {
         const today = new Date();
         const utcOffset = today.getTimezoneOffset();
-        const baseApiUrl = `https://europe-west1-tweetedonthisday.cloudfunctions.net/getTweetsOnThisDay?username=${username}&utcOffset=${utcOffset}`;
-        let baseYear = today.getFullYear() - 1;
+        const baseApiUrl = `https://europe-west1-oldtweetstoday.cloudfunctions.net/getTweetsOnThisDay?username=${username}&utcOffset=${utcOffset}&day=${day}&month=${month}`;
```

Comment:
I'm deploying the updated backend, so don't forget to update this url.

Reply:
Thanks for the reminder. Done